serve/mlc_serve/model/paged_cache_manager.py

```diff
@@ -104,18 +104,18 @@ def replace_head_prompt_block_with(self, new_block):
 class KVCache:
     def __init__(
         self,
-        cache_blocks,
-        block_size,
+        cache_blocks: Any,
+        block_size: int,
     ):
         self.cache_blocks = cache_blocks
         self.block_size = block_size
 
         # SequenceId -> list[int]
-        self.prompt_block_tables = defaultdict(list)
-        self.slot_mappings = defaultdict(list)
+        self.prompt_block_tables = defaultdict(list)  # type: ignore
+        self.slot_mappings = defaultdict(list)  # type: ignore
 
         # The core data structure
-        self.decode_block_tables = dict[SequenceId, DecodeBlockTable]()
+        self.decode_block_tables: dict = dict[SequenceId, DecodeBlockTable]()
```

Comment:
Is this needed (seems odd)? I'm aware that mypy can give different outputs for different versions.

Reply:
It is needed for mypy